### ISSUE TITLE
Workflow Golden team- VC https://github.com/eliasgkh/vc/projects

### ISSUE THREAD

**Issue by Cynthia-aboumaroun** [projets2017cl/vc#80]
Bonjour Dr. Pascal,

Nous avons crée un groupe de travail de 5 members sous le nom de "Workflow Golden team"; 1- Aless Hossry 2- Cynthia Abou Maroun 3- Edgard Chamoun 4- Elias Ghazal 5- Elias Khoury
et ds votre nouvelle page https://projets2017cl.github.io/vc/sousProjets, on n'existe pas. qu' est ce qu'on doit faire???
merci

**Comment by pascalfares** [comment]
Pourquoi vous ne rejoingner pas https://github.com/projets2017cl/WorkflowProject ?

@t-g-tannous @Cynthia-aboumaroun ?

En tout cas pour modifier ces pages  https://projets2017cl.github.io/vc

faire fork de 

1- du projet vc
2- Faire vos modifications dans le répertoire /docs
3- me demander un pullrequest

@projets2017cl/vc

**Comment by eliasgkh** [comment]
Bonjour Dr. Pascal,

j'ai besoin de votre aide : 
j'ai fait fork sous Valeurc2017/vc au lieu de la faire sous Projets2017/vc comment puis-je regler ce probleme.
 Merci d'avance.
Cordialement.
![vc](https://user-images.githubusercontent.com/22739525/28489307-cf667894-6ec9-11e7-8f7b-9e48b2295267.PNG)

**Comment by pascalfares** [comment]
Tu as fait des changement dans eliasgkh/vc?

**Comment by eliasgkh** [comment]
Oui, j’ai déjà ajouté les Collaborateurs et créer plusieurs issue et un projet.
